roles/nginx/templates/alfresco_proxy.include.j2

```diff
@@ -44,24 +44,32 @@
         include {{ nginx_vhost_path }}/alfresco_proxy_headers.include;
     }
 
+{% if groups['identity'] | default([]) | length != 0 %}
```

Comment:
> will only add the nginx location into the config file if the components are really installed

thanks, something which is good to have!

Would you mind changing the length checks to `length > 0` which should more consistent?

Reply:
No problem, it's done. Sometimes the `groups['repository']` is used and sometimes it's `groups.repository`. Not sure if you would like to align all of them as well at some point.

Reply:
I don't have strong opinions on this but better be consistent